addons/packages/harbor/2.2.3/bundle/config/values.yaml

```diff
@@ -1,6 +1,3 @@
-#@data/values
```

Comment:
I ran `ytt -f addons/packages/harbor/2.2.3/bundle/config/` and got this error:

```shell
ytt: Error: Evaluating starlark template: 
- struct has no .namespace field or method
    in validate_harbor_namespace
      6 |   values.namespace or assert.fail("harbor namespace should be provided")
    in validate_harbor
      101 |      validate_func()
    in <toplevel>
      109 | validate_harbor()
```

I think that you want to keep the `#@data/values` at the top here to indicate that this  is a data values file.

Reply:
Have you tried installing it based on the documentation and not ytt? I'm going by what the documentation says on installing Harbor and that's the error I received.

Reply:
I agree with @kacole2 that this does fix the issue but, I don't think it's the right fix.  The other packages include those two lines in their values.yaml as well.

Reply:
I don't think this fix is correct. We will start to update the doc to resolve the problem when installing the harbor package today.

Reply:
We've already updated the documentation, pls refer to the [#PR2648](https://github.com/vmware-tanzu/community-edition/pull/2648).